e2e/rpc/rpc_test.go

```diff
@@ -146,6 +146,26 @@ func (s *RPCTestSuite) TestCallRawResult() {
 	s.NodeManager.StopNode() //nolint: errcheck
 }
 
+// TestCallRawResultGetTransactionReceipt checks if returned response
+// for a not yet mained transaction is null.
+// Issue: https://github.com/status-im/status-go/issues/547
+func (s *RPCTestSuite) TestCallRawResultGetTransactionReceipt() {
```

Comment:
but the test actually passed without a fix, maybe there is some way to simulate an error?

Reply:
Yeah, I was not able to actually reproduce it. I suspect that there may be an error during marshaling, thus added a code to `call_raw.go`.

Reply:
that won't hurt for sure